Code/JavaWrappers/ROMol.i

```diff
@@ -155,6 +156,18 @@ void setPreferCoordGen(bool val) {
 bool getPreferCoordGen();
 void setPreferCoordGen(bool);
 
+%{
+bool getUseLegacyStereoPerception() {
+  return RDKit::Chirality::getUseLegacyStereoPerception();
+}
+void setUseLegacyStereoPerception(bool val) {
+  RDKit::Chirality::setUseLegacyStereoPerception(val);
+}
+%}
+
+bool getUseLegacyStereoPerception();
+void setUseLegacyStereoPerception(bool);
+
```

Comment:
Let's expose the non-tetrahedral chirality flag at the same time:
```suggestion
%{
bool getUseLegacyStereoPerception() {
  return RDKit::Chirality::getUseLegacyStereoPerception();
}
void setUseLegacyStereoPerception(bool val) {
  RDKit::Chirality::setUseLegacyStereoPerception(val);
}
bool getAllowNontetrahedralChirality() {
  return RDKit::Chirality::getAllowNontetrahedralChirality();
}
void setAllowNontetrahedralChirality(bool val) {
  RDKit::Chirality::setAllowNontetrahedralChirality(val);
}
%}

bool getUseLegacyStereoPerception();
void setUseLegacyStereoPerception(bool);
bool getAllowNontetrahedralChirality();
void setAllowNontetrahedralChirality(bool);
```

Reply:
@greglandrum I have applied your suggestions, and also exposed the same functionality in MinimalLib and CFFI (with tests).